tangelo/linq/tests/test_translator_circuit.py

```diff
@@ -236,6 +247,11 @@ def test_cirq(self):
 
         np.testing.assert_array_equal(v1, v2)
 
+        job_sim = cirq_simulator.simulate(translated_circuit2)
```

Comment:
These tests have become confusing overall. One test function is now a bunch of copy pastes that blend together. We don't need to repeat the same comments every time, we should now introduce comments that say what the next 10 lines are doing etc to clearly delineate the "sub tests" that you are running here.

qulacs just repeats the same blocks and comments on and on, with no clear structure. cirq has no comments at all.

Reply:
Our translation of qiskit does not need to be changed. It currently does not support more than single controls at this point. Previously, I believe that qiskit just had multi-controlled X gates but they have a few others now. Maybe something to look into.

Reply:
Thank you. We do not need to look into this actively, there's no clear demand